docs/ilm.asciidoc

```diff
@@ -49,14 +49,42 @@ readonly,
 IMPORTANT: Changes to the default index lifecycle policy do not take effect until the current index has rolled over.
 If you'd like to manage a custom policy, see <<manual-ilm-setup,Manual ILM>>.
 
+[float]
+[[ilm-enable]]
+==== Enable ILM
+
+ILM can be enabled/disabled via the `ilm.enabled` configuration option.
+Starting in v7.3, `ilm.enabled` is set to `"auto"` by default.
+
+[float]
+===== `ilm.enabled: "auto"`
+
+`"auto"` means APM Server will attempt to enable ILM based on the following conditions:
+
+* *`output.elasticsearch` must be enabled*. If any other output is configured, ILM will be disabled and an error will be logged.
+* *Custom `index` or `indices` settings cannot be configured*. If custom index settings are configured,
+ILM will be disabled, as it only works with default index settings.
+* *The Elasticsearch instance must support ILM*. If it does not, ILM will be disabled.
```

Comment:
I would move this up to be the condition right after `output.elasticsearch` must be enabled.